plugins/woocommerce/includes/admin/helper/class-wc-helper-updater.php

```diff
@@ -272,21 +274,37 @@ public static function display_notice_for_expired_and_expiring_subscriptions( $p
 		// Prepare the expiry notice based on subscription status.
 		$expiry_notice = '';
 		if ( ! empty( $subscription['expired'] ) && ! $subscription['lifetime'] ) {
+
+			$renew_link = add_query_arg(
+				array(
+					'utm_source'   => 'pu',
+					'utm_campaign' => 'pu_plug_screen_renew',
```

Comment:
maybe this should be `pu_plugin_screen_renew`?

Reply:
Updated with https://github.com/woocommerce/woocommerce/pull/49645/commits/d048db42d574aade5a61664dd4093f8e835cc734